configuration.yaml

```diff
@@ -1,10 +1,15 @@
 global:
   logDir: gs://otar000-evidence_input/parser_logs
   cacheDir: cache_dir
-  schema: https://raw.githubusercontent.com/opentargets/json_schema/2.4.0
+  schema: https://raw.githubusercontent.com/opentargets/json_schema/master
```

Comment:
This works and robust, however when preparing for release I think will be nicer to refer for a specific release from the json_schema repo.